binary_search_tree/binary_search_tree.py

```diff
@@ -10,13 +10,41 @@ def __init__(self, value):
     self.right = None
 
   def insert(self, value):
-    pass
+    if value < self.value:
+      if not self.left:
+        self.left = BinarySearchTree(value)
+      else:
+        self.left.insert(value)
+    else:
+      if not self.right:
+        self.right = BinarySearchTree(value)
+      else:
+        self.right.insert(value)
 
   def contains(self, target):
-    pass
+    if target == self.value:
+      return True
+    elif target < self.value:
+      if self.left:
+        return self.left.contains(target)
+      else:
```

Comment:
You can also simplify this function to only have one `return False` statement. Can you think of how?